<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundPhoto">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>650</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Media/Des2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="howItWorksHeadText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>128</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 70 18pt &quot;Century Gothic&quot;; 
color: black;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Upload File to Visualize Actions</string>
   </property>
  </widget>
  <widget class="QLabel" name="howItWorksText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 70 12pt &quot;Century Gothic&quot;; 
color: black;</string>
   </property>
   <property name="text">
    <string>Select the folder with annotated data and videos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 15px;

	background-color: rgb(255, 255, 255);

	font: 75 14pt; color: rgb(0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: rgb(224, 224, 224);
	font: 70 13pt &quot;Century Gothic&quot;; 
   color: black;
}

QPushButton:hover{
	background-color: rgb(73, 96, 96);
}
QPushButton:pressed{
	background-color: black;
}</string>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FACButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: rgb(224, 224, 224);
	font: 70 11pt &quot;Century Gothic&quot;; 
   color: black;
}

QPushButton:hover{
	background-color: rgb(73, 96, 96);
}
QPushButton:pressed{
	background-color: black;
}</string>
   </property>
   <property name="text">
    <string>Go to main page</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
